data-raw/qs_metabolic.R

```diff
@@ -62,19 +60,34 @@ qs_metabolic_shell <- visit_schedule %>%
 qs_metabolic_results <- qs_metabolic_shell %>%
   mutate(
     DOMAIN = "QS",
-    QSORRES = round(runif(n(), min = 0, max = 100)),
-    QSORRESU = "cm",
-    QSSTRESU = "cm",
-    QSSTRESN = QSORRES,
-    QSSTRESC = as.character(QSORRES),
+    QSORRES = as.character(round(runif(n(), min = 0, max = 100))),
+    QSORRESU = "mm",
+    QSSTRESU = "mm",
+    QSSTRESN = as.numeric(QSORRES),
+    QSSTRESC = QSORRES,
     QSBLFL = if_else(VISIT == "BASELINE", "Y", NA_character_)
+  ) %>%
+  # Update the free text question (COEQ20)
+  mutate(
+    QSSTRESC = case_when(
+      QSTESTCD == "COEQ20" & QSSTRESN < 33 ~ "Ice Cream",
+      QSTESTCD == "COEQ20" & QSSTRESN < 66 ~ "Pizza",
```

Comment:
What if `QSSTRESN` for `COEQ18` [Savoury foods (french fries, crisps, burgers, pizza, etc)] turns out to be zero?

Reply:
Haha fair point. Luckily, for this seed, it never is 😄 
![image](https://github.com/user-attachments/assets/5e233536-dec2-4f79-afd1-8f30a1df7f17)